README.md

```diff
@@ -105,6 +105,15 @@ let public_key = SignedPublicKey::from_string(&key_string).unwrap().0;
 public_key.verify_signature(HashAlgorithm::default(), &data[..], &new_signature).unwrap();
 ```
 
+## Experimental Cryptography
```

Comment:
I would prefer a section called `Features`, listing the availble features (with explanations), but no need for you to write them, I can do that as a follow up

Reply:
I've re-scoped this proposed section in `README.md`, but of course i don't mind if you want to modify it however you think is useful.

Reply:
thanks